twisted/test/test_compat.py

```diff
@@ -315,6 +317,26 @@ def test_python3(self):
 
 
 
+class PYPYTest(unittest.SynchronousTestCase):
+    """
+    Identification of PyPy.
+    """
+
+    def test_pypy(self):
+        """
+        On PyPy, C{_PYPY} is True.
+        """
+        if 'PyPy' in sys.version:
+            self.assertTrue(_PYPY)
+
```

Comment:
I would still prefer to keep the two as separate tests so that a failure in either is immediately obvious -- I think this is why `PY3Tests` chose this approach.